src/components/ChangePlan/Checkout/Checkout.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import HeaderSection from '../../shared/HeaderSection/HeaderSection';
+import { useIntl } from 'react-intl';
+import { Loading } from '../../Loading/Loading';
+import { Helmet } from 'react-helmet';
+import { ContactInformation } from './ContactInformation/ContactInformation';
+import { getCountries } from '../../../utils';
+
+const Checkout = ({ dependencies: { dopplerUserApiClient } }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const selectOption = _('checkoutProcessForm.empty_option_select');
+
+  const checkoutSteps = {
+    contactInformation: 'contact-information',
+    billingInformarion: 'billing-information',
+    paymentInformarion: 'payment-information',
+  };
+
+  const [activeStep, setActiveStep] = useState(checkoutSteps.contactInformation);
+  const [state, setState] = useState({ loading: true });
+
+  const submitContactInformationForm = async (values) => {
+    const result = await dopplerUserApiClient.createOrUpdateContactInformation(values);
+
+    if (result.success) {
+      setActiveStep(checkoutSteps.contactInformation);
+    }
+
+    return result;
+  };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const contactInformationResult = await dopplerUserApiClient.getContactInformationData();
+      const defaultValue = { key: '', value: selectOption };
+      const countries = [defaultValue, ...getCountries(intl.locale)];
+
+      setState({
+        contactInformation: contactInformationResult.success
+          ? contactInformationResult.value
+          : null,
+        isError: !contactInformationResult.success,
```

Comment:
Better without is, it sounds weird isError.
```suggestion
        error: !contactInformationResult.success,
```

Reply:
Mmm, but error means _an error_ not _has an error_.

What do you think about keep `success` or `hasError`?

Reply:
I will keep: ```success```